content/en/architectures/adobe/index.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Scaling Adobe’s Service Delivery Foundation with a Cell-based Architecture
+date: 2024-10-11
+org_name: Adobe
+org_team: Developer Platforms
+org_url: https://www.adobe.com/
+contact: Vikram Sethi
```

Comment:
```suggestion
org_url: https://www.adobe.com/
org_logo_filename: <adobe.svg or adobe.png>
contact: Vikram Sethi
```
To add a logo, you can put it in the same directory with the other images and reference it with `org_logo_filename` and it will appear next to the metadata about the reference architecture :)